modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -15,7 +15,7 @@ The cluster administrator can use this performance profile configuration to make
 
 [NOTE]
 ====
-The RT kernel is only supported on worker nodes.
+In most deployments, kernel-rt is supported only on worker nodes when you use a standard cluster with three master nodes and three worker nodes. There are exceptions for compact and single nodes on OpenShift deployments. For installations on a single node, kernel-rt is supported on the single control-plane node.
```

Comment:
Couple of small nits:
```suggestion
In most deployments, kernel-rt is supported only on worker nodes when you use a standard cluster with three control plane nodes and three worker nodes. There are exceptions for compact and single nodes on {product-title} deployments. For installations on a single node, kernel-rt is supported on the single control-plane node.
```
FWIW, we do often refer to kernel-rt as "real-time kernel" in our docs. But considering that this has already been reviewed and changed, I'm happy with the way it is spelled here. (There's precedent for both ways in both vanilla docs and in `cnf-` files.